internal/cmd/asyncapi/account_details.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/confluentinc/cli/internal/pkg/cmd"
```

Comment:
This is an internal import so it shouldn't be in the same group as the standard imports. Your IDE should be able to sort this list for you automatically!